openllm-core/src/openllm_core/prompts/__init__.py

```diff
@@ -0,0 +1 @@
+from __future__ import annotations
```

Comment:
let's just expose all API on `__init__.py` top-level like this. Then you can do `from openllm_core.prompt import process_prompt, PromptTemplate`

```suggestion
from __future__ import annotations

from ._prompt import process_prompt
from .prompt_template import PromptTemplate
```